python/deltalake/table.py

```diff
@@ -607,6 +614,35 @@ def to_pandas(
             filters=filters,
         ).to_pandas()
 
+    def to_ray(
+        self,
+        filesystem: Optional["pyarrow.fs.FileSystem"] = None,
+        columns: Optional[List[str]] = None,
+        parallelism: int = -1,
+    ) -> "ray.data.dataset.Dataset":
+        from ray.data import read_parquet
+        from ray.data.datasource import DefaultParquetMetadataProvider
+
+        meta_provider = DefaultParquetMetadataProvider()
+        """Create an Arrow dataset from a Delta Table using Ray
+
+        Args:
+            filesystem: filesystem implementation to read from
+            columns: list of columns to read
+            parallelism: requested parallelism of read, may be limited by number of files
+
+        Returns:
+            Dataset holding Arrow records read from the Delta Lake Table
+        """
+
+        return read_parquet(
+            paths=self.file_uris(),
```

Comment:
I'd be reticent to merge this, because this isn't future-proof to new versions of the Delta Lake format. The idea that a Delta Table is just a list of Parquet files is outdated, given features like [column mapping](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#column-mapping) and [deletion vectors](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#deletion-vectors).

If you do want to go through with this, then we at least need to place a guard limiting the protocol `minReaderVersion` to 1.

Reply:
sure , will add guard against this.

Reply:
added